components/org.wso2.carbon.identity.account.suspension.notification.task/src/main/java/org/wso2/carbon/identity/account/suspension/notification/task/ldap/LDAPNotificationReceiversRetrieval.java

```diff
@@ -145,4 +148,45 @@ public void init(RealmConfiguration realmConfiguration) {
         return users;
     }
 
+    /* Convert Active Directory date format (Generalized Time) to WSO2 format.
+     *
+     * @param date Date formatted in Active Directory date format.
+     * @return Date formatted in WSO2 date format.
+     */
+    private long convertDateFormat(String date) {
```

Comment:
IMO, this method name should be more self-explanatory. Ex:
- getDateInWSO2Format
- convertToWSO2DateFormat